### DISCUSSION TITLE
Tracker not visible wehn working on a task?

### DISCUSSION THREAD

**Discussion by niels-przybilla** [kimai/Paid-plugins#147] [Questions & Answers] [answered]
Hello,

the time on the top is not starting to count when I started a timer on a task?

Many thanks for help to understand.

Best wiishes

Niels
![Monosnap Kimai – Zeiterfassung 2024-04-16 09-48-02](https://github.com/kevinpapst/kimai-plugins/assets/99684821/c9553ee6-9493-4d43-be3a-a32da36379c6)

**Comment by pbksol** [comment]
The shown company is not a real customer or your own company? Just asking – you should obfuscate screenshots to not publish customer data.

Which version of Kimai are you using? And which version of the Tasks plugin? I tried it on our own instance and everything works fine.

**Comment by niels-przybilla** [comment]
Hi Pascal.

good point with the screenshot. Thank you.

Kimai 2.13.0
Task Plugin: 2.5.0

Best wishes

Niels

**Comment by pbksol** [comment, marked as answer]
Wild guess… Could it be that the task is assigned to a user and you are logged in with another (admin)user?

If you then start the task processing, the timer is started in the context of the assigned user.

**Reply by niels-przybilla** [reply]
You got me :-) I am still testing and I have an admin and regular user and i selected the regular user and I am logged in as admin ! THX
